lib/fika/cli.ex

```diff
@@ -46,7 +46,7 @@ defmodule Fika.Cli do
     try do
       Logger.debug("Calling :#{module}.#{function}")
       {result, _binding} = Code.eval_string(~s':"#{module}".#{function}')
-      IO.inspect(result)
+      result |> inspect() |> IO.puts()
```

Comment:
Is this change required?

Reply:
Not really, but since this is an intentional print, it makes debuggin `IO.inspect`s easier to find when commiting